charts/telemetry-server/values.yaml

```diff
@@ -6,9 +6,9 @@ replicaCount: 1
 
 image:
   repository: docker.io/gbuenodevsuse/telemetry-server #until we have an image built in our build system
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
   # Overrides the image tag whose default is the chart appVersion.
-  tag: latest
+  tag: test2
```

Comment:
Did you mean to make this change, and the pullPolicy setting change, permanent?

Reply:
Changed tag back to latest, as for the pullPolicy, it makes sense to leave it as Always.